napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -227,11 +227,11 @@ def changeRendering(self, text):
               This will make nearer objects appear more prominent.
         """
         self.layer.rendering = text
-        self._toggle_rendering_parameter_visbility()
+        self._update_rendering_parameter_visibility()
 
     def changeDepiction(self, text):
         self.layer.depiction = text
-        self._toggle_plane_parameter_visibility()
+        self._update_plane_parameter_visibility()
```

Comment:
I made these changes to clarifying that this is updating based on other state, rather than a true toggle of the plane control visibility.